binding/ios/CobraAppTest/PerformanceTest/PerformanceTest.swift

```diff
@@ -23,7 +23,7 @@ class PerformanceTest: XCTestCase {
     }
 
     func testPerformance() throws {
-        try XCTSkipIf(thresholdString == "{PERFORMANCE_THRESHOLD_SEC}")
+        try XCTSkipIf(thresholdString == nil)
```

Comment:
I think this was set to `{PERFORMANCE_THRESHOLD_SEC}` so that if you didn't replace the `thresholdString` default value, it would skip the test. I think we still need that otherwise it will run the test by default unless you explicitly set it to nil.

Reply:
I reverted this, although I'm a bit confused - the `Inject Performance Threshold` step in the actions seemed to replace all instances of `{PERFORMANCE_THRESHOLD_SEC}`, causing the test to be skipped - not sure how this passed in the past. I changed it though to only replace the first instance.